src/core/core-varlink.c

```diff
@@ -573,6 +573,40 @@ static int vl_method_describe(sd_varlink *link, sd_json_variant *parameters, sd_
         return sd_varlink_replybo(link, SD_JSON_BUILD_PAIR_VARIANT("manager", v));
 }
 
+static int vl_method_describe_jobs(sd_varlink *link, sd_json_variant *parameters, sd_varlink_method_flags_t flags, void *userdata) {
+        Manager *m = ASSERT_PTR(userdata);
+        Job *j;
+        int r;
+
+        assert(parameters);
+
+        if (sd_json_variant_elements(parameters) > 0)
+                return sd_varlink_error_invalid_parameter(link, parameters);
```

Comment:
i'd prefer if we optionally take an input parameter here: a job id. if set we'll only return info about that. the importd, machined, bootctl list apis might serve as inspirations there.